spec/activefedora/ordered_spec.rb

```diff
@@ -62,6 +61,27 @@ class Image < ActiveFedora::Base
         expect(subject.ordered_member_proxies.to_a.length).to eq 1
       end
     end
+    describe "#=" do
+      it "can set ordered members" do
```

Comment:
I have a preference for saying `it "sets ordered members"`. Using "can" here is akin to using "should".